### PR TITLE
Added 5minselfcare twitter bot

### PR DESCRIPTION
**Pull request by trymnilsen**
Added another bot from the fantastic https://twitter.com/5minselfcare

### CODE REVIEW

**Comment by notwaldorf** [comment]
Ah, this actually won't work, since I only look at the first 3 bots. Maybe in the first README paragraph we can start a "Other bots you might like: " sentence?

File: config.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 var config = {};
 // Accounts to read the last tweet from. The first one in the list will be
 // spoken by the party parrot.
-config.twitter = process.env.TTC_BOTS || 'tinycarebot,selfcare_bot,magicrealismbot';
+config.twitter = process.env.TTC_BOTS || 'tinycarebot,selfcare_bot,magicrealismbot,5minselfcare';
```
Comment (mojoaxel) [inline config.js]: I would prefer the `5minselfcare` over ...lets say.... the `magicrealismbot`
@trymnilsen Maybe you want to change your PR?

Comment (notwaldorf) [inline config.js]: you can configure which bots you want to use with the TTC_BOTS env variable! See the readme for instructions

**Comment by trymnilsen** [comment]
Agree, suggesting some other nice bots via a sentence in the readme and letting them choose themselves might be the best idea.

**Pull request closed without merging**